plugin/al/plugin/AL_USDMayaTestPlugin/py/testProxyShape.py

```diff
@@ -48,8 +48,7 @@ def tearDownClass(cls):
 
     def setUp(self):
         unittest.TestCase.setUp(self)
-        self.workingDir = os.path.abspath(type(self).__name__)
-        os.mkdir(self.workingDir)
+        self.workingDir = os.environ.get('MAYA_APP_DIR')
```

Comment:
Don't write temporary testing files to the source tree. Instead use the `MAYA_APP_DIR` which is a different temp folder created for each unit test.